src/functions/genfrac.js

```diff
@@ -319,6 +317,31 @@ defineFunction({
     mathmlBuilder,
 });
 
+defineFunction({
+    type: "genfrac",
+    names: ["\\cfrac"],
+    props: {
+        numArgs: 2,
+    },
```

Comment:
I didn't realize that `\cfrac` wasn't allowed as an argument, but it makes sense that it isn't.